src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/SecretUtils.java

```diff
@@ -62,6 +66,8 @@ public abstract class SecretUtils {
 
     static final String JENKINS_IO_CREDENTIALS_SCOPE_LABEL = "jenkins.io/credentials-scope";
 
+    /** Optional annotation containing a list of job folders this credential is available to */
+    static final String JENKINS_IO_CREDENTIALS_ITEM_GROUP_ANNOTATION = "jenkins.io/credentials-item-group";
```

Comment:
open for discussion on a better annoation name

Reply:
naming is hard :)

`jenkins.io/credentials-store-location` (`s`) ?